yelp_package/extra_requirements_yelp.txt

```diff
@@ -1,22 +1,41 @@
 --index-url=https://pypi.yelpcorp.com/simple
+atomicfile==1.0
 cached-property==1.3.1
+cffi==1.15.0
+cffi==1.15.0
 clusterman_metrics==2.2.0
-crypto_lib==4.0.0
+crypto-lib==4.0.0
+cryptography==2.8
```

Comment:
is there a reason that we have to use such an old version? It might be worth ticketing this up, I'm actually surprised such an old version would work with newer libssl versions

Reply:
i get the feeling that we just bump it whenever it causes an issue - lemme see what we can bump up to

Reply:
3.0 works! (and is the highest we can bump to atm)